src/proof/trust_id.h

```diff
@@ -66,6 +66,10 @@ enum class TrustId : uint32_t
   QUANTIFIERS_PREPROCESS,
   /** A subtype elimination step that could not be processed */
   SUBTYPE_ELIMINATION,
+  /** A rewrite required for showing a macro theory rewrite */
+  MACRO_THEORY_REWRITE_RCONS,
```

Comment:
What is this for?

Reply:
This will be used if elaboration of a `ProofRewriteRule::MACRO_` fails.  This is not expected to ever happen.